next.json

```diff
@@ -280,6 +280,13 @@
                 ]
             },
             {
+                "name": "set stencil reference",
+                "args": [
+                    {"name": "back reference", "type": "uint32_t"},
```

Comment:
Like you mentioned, D3D12 can't set the front and back stencil refs separately, so we should no advertise that option.